packages/playground-server/README.md

```diff
@@ -37,3 +37,31 @@ Instead of using the regular PostgreSQL database, the test scope uses a volatile
 ## Environment settings
 
 Unlike other packages, the Server relies on a [`.env-cmdrc`](./.env-cmdrc) which allows to configure multiple environments in the same file. See [env-cmd](https://www.npmjs.com/package/env-cmd#rc-file-usage)'s reference for more information on how it works.
+
+## Funding the Hub Account for Playground Testing
+
+### Where do I send Kovan ETH for testing?
+
+First of all, you need to generate a mnemonic:
+
+```node
+$ node
+> require("ethers").Wallet.createRandom().mnemonic
+'camera enter drive paper elegant camp above attend board thought inch crash'
+```
+
+And save it in `.env.local`.
+
+**Option 1** Compute it:
```

Comment:
This `it` and the `it` above refer to two different things.

"And save the mnemonic in `.env.local`"

"***Option 1** Compute the address"